internal/yggdrasil/yggdrasil_test.go

```diff
@@ -586,6 +592,643 @@ var _ = Describe("Yggdrasil", func() {
 			Expect(eventsRecorder.Events).To(Receive(ContainSubstring("Auth file secret")))
 		})
 
+		It("Secrets reading failed", func() {
+			// given
+			deviceName := "foo"
```

Comment:
This deviceName&device can be defined on the `beforeaEach` section.

Reply:
There's no need